src/Platforms/MySQL/Comparator.php

```diff
@@ -71,16 +74,17 @@ private function normalizeTable(Table $table): Table
                 continue;
             }
 
+            /** @phpstan-ignore argument.type */
```

Comment:
All elements of `PlatformOptions` are optional, so subtracting the keys of `$tableOptions` from a `PlatformOptions` value via `array_diff_assoc()` will always produce another `PlatformOptions` but PHPStan doesn't understand that.